definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -2763,6 +2763,18 @@
     "RetriesTimeoutURLSubmittedEvent": "120,120",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "05/02/2021",
+    "CaseTypeID": "NO_FAULT_DIVORCE",
+    "ID": "draftUpdate",
```

Comment:
As this is a new event you will also need entry in Authorisation Case Event and also CaseEventToFields tab

Reply:
Added authorisations and case event to field entry